lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -103,7 +104,9 @@ defmodule ExUnit.Runner do
 
       # Slots are available, start with async modules
       modules = ExUnit.Server.take_async_modules(available) ->
-        running = spawn_modules(config, modules, running)
+        tests_per_file = tests_per_file(ExUnit.Server.get_all_modules())
```

Comment:
Unfortunately this is problematic because modules are loaded dynamically. So there is a chance I have two modules in the same file, the first one is loaded, the second one is not yet loaded, so `get_all_modules` won't return the second one and still run more tests than it should. :(

Reply:
Unfortunately I can't come up with any other ideas either. the best option i can think of is to look into the test manifest and tranform `line: ...` into `line: ..., max_line: ...`. But even then is a bit tricky.

Reply:
I get this; I just wasn't sure if maybe all test cases from one file are loaded at once. That should solve this, but it would still be pretty ugly because it would be very implicit.

Another thought I had (which might be what you're saying) is to do it independently from the current `line` tag (so keep the `line` tag behavior as it is).
And to introduce a set of `min_line` and `max_line,` and execute all the tests from the line range if both are passed.
So something similar to the first attempt at solving this, but independent from the current behavior and in mix, this could be passed, like `mix test test_file.exs:10-30`

Reply:
> So something similar to the first attempt at solving this, but independent from the current behavior and in mix, this could be passed, like mix test test_file.exs:10-30

Oh, I like this approach! It may give additional features too.